frontend/src/Dashboard/CrossingListPage/CrossingListHeader/CrossingListHeader.js

```diff
@@ -25,26 +35,26 @@ class CrossingListHeader extends Component {
 
             {params.smallsize ? (
               <div className='smallflex'>
-                <div className={classnames(params, 'CrossingListSortToggle')}>
+                <div className={classnames(params, 'CrossingListSortToggle')} onClick={this.toggleSortDirection}>
                   <div className={classnames(params, 'CrossingListSortToggleText')}>
-                    LAST UPDATED <FontAwesome name="caret-up" />
+                    LAST UPDATED {this.state.invertSort ? <FontAwesome name="caret-up" /> : <FontAwesome name="caret-down" />}
                   </div>
                 </div>
-                <div className={classnames(params, 'CrossingListFilterToggle')}>
+                <div className={classnames(params, 'CrossingListFilterToggle', {'selected': this.state.showFilterDrawer})} onClick={this.toggleFilterDropdown}>
                   <div className={classnames(params, 'CrossingListFilterToggleText')}>
-                    FILTER <FontAwesome name="plus" />
+                    FILTER {this.state.showFilterDrawer ? <FontAwesome name="minus" /> : <FontAwesome name="plus" />}
                   </div>
                 </div>
               </div>
             ) : (
-              <div className={classnames(params, 'CrossingListSortToggle')}>
+              <div className={classnames(params, 'CrossingListSortToggle')} onClick={this.toggleSortDirection}>
                 <div className={classnames(params, 'CrossingListSortToggleText')}>
-                  LAST UPDATED <FontAwesome name="caret-up" />
+                  LAST UPDATED {this.state.invertSort ? <FontAwesome name="caret-up" /> : <FontAwesome name="caret-down" />}
```

Comment:
Small suggestion here. Line 52 looks a lot like 45. And 40 is the exact same. This repetition is a clue that we could abstract these one line ternaries into a `const` set outside of render.

Reply:
I looked into ways to reduce copypasta here and found that putting the smallsize versions in a div was causing issues with conditional rendering. I think a decent future option for this instead of moving those lines out into consts would be to move the toggle controls out into their own components. I added an issue for that: https://github.com/cityofaustin/ctxfloods/issues/60